src/plugins/intel_cpu/src/nodes/strided_slice.cpp

```diff
@@ -38,8 +39,113 @@ bool StridedSlice::isSupportedOperation(const std::shared_ptr<const ov::Node>& o
     return true;
 }
 
+inline int calculate_output_shape(const int& startIdx, const int& stopIdx, const int& step, const int& len) {
+    auto start = (startIdx > len - 1) ? len - 1 : ((startIdx % len + len) % len);
+    auto stop = (stopIdx > len - 1)   ? len     : ((stopIdx % len + len) % len);
```

Comment:
Is not try to to normalize and clip the start/stop to dimension like [here](https://github.com/openvinotoolkit/openvino/blob/b8348cda2e992b2e163a2c326013e59688efbb23/src/core/shape_inference/include/slice_shape_inference_utils.hpp#L164)
maybe re-use these functions, they use simple operation like compare,add no `%`.